docs/Targets.md

```diff
@@ -1572,3 +1572,87 @@ Flash Allocation:
 ```
 
 Detailed steps can be found at [Readme](../IDE/Renesas/e2studio/RA6M4/Readme.md).
+
+
+## Intel x86_64 with Intel FSP support
```

Comment:
Can you include sample boot output?

Reply:
working on that: other parts of the documentation need to be updated